chapters/operatorsandexpressions.tex

```diff
@@ -1317,19 +1317,16 @@ \subsection{Parameter Expressions}\doublelabel{parameter-expressions}
   \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with parameter
   subexpressions is a parameter expression.
 \item
-  Some function calls are parameter expressions even if the arguments
-  are not:
-
+  Some function calls are parameter expressions even if the arguments are not:
```

Comment:
as far as I understand this is an unrelated formatting change. I couldn't see anything related to expandable connectors here.

Reply:
I'm afraid the commit message is all I have to say:
> Tidy up function calls that are parameter expressions
>
> Stumbled upon these when looking for specification of 'size'.

Reply:
Fixed via #2606.  Resolving.